docs/csharp/language-reference/builtin-types/nullable-value-types.md

```diff
@@ -0,0 +1,131 @@
+
+At run time, if the value of a nullable value type is `null`, the explicit cast throws an <xref:System.InvalidOperationException>.
+
+A non-nullable value type `T` is implicitly convertible to the corresponding nullable value type `T?`.
+
+## Lifted operators
+
+The predefined unary and binary operators or any overloaded operators that are supported by a value type `T` are also supported by the corresponding nullable value type `T?`. These operators, also known as *lifted operators*, produce `null` if one or both operands are `null`; otherwise, the operator uses the contained values of its operands to calculate the result. For example:
+
+[!code-csharp[lifted operators](~/samples/csharp/language-reference/builtin-types/NullableValueTypes.cs#LiftedOperator)]
+
+> [!NOTE]
+> For the `bool?` type, the predefined `&` and `|` operators don't follow the rules described in this section: the result of an operator evaluation can be non-null even if one of the operands is `null`. For more information, see the [Nullable Boolean logical operators](../operators/boolean-logical-operators.md#nullable-boolean-logical-operators) section of the [Boolean logical operators](../operators/boolean-logical-operators.md) article.
+
+For the [comparison operators](../operators/comparison-operators.md) `<`, `>`, `<=`, and `>=`, if one or both operands are `null`, the result is `false`; otherwise the contained values of operands are compared. Do not assume that because a particular comparison (for example, `<=`) returns `false`, the opposite comparison (`>`) returns `true`. The following example shows that 10 is
+
+- neither greater than or equal to `null`
+- nor less than `null`
+
+[!code-csharp-interactive[relational and equality operators](~/samples/csharp/language-reference/builtin-types/NullableValueTypes.cs#ComparisonOperators)]
+
+The preceding example also shows that an equality comparison of two nullable value type instances that are both `null` evaluates to `true`.
+
+If there exists a [user-defined conversion](../operators/user-defined-conversion-operators.md) between two value types, the same conversion can also be used between the corresponding nullable value types.
+
+## Boxing and unboxing
+
+An instance of a nullable value type `T?` is [boxed](../../programming-guide/types/boxing-and-unboxing.md) as follows:
+
+- If <xref:System.Nullable%601.HasValue%2A> returns `false`, the null reference is produced.
+- If <xref:System.Nullable%601.HasValue%2A> returns `true`, the corresponding value of the underlying value type `T` is boxed, not the instance of <xref:System.Nullable%601>.
+
+You can unbox a boxed value of a value type `T` to the corresponding nullable value type `T?`, as the following example shows:
+
+[!code-csharp-interactive[boxing and unboxing](~/samples/csharp/language-reference/builtin-types/NullableValueTypes.cs#Boxing)]
+
+## How to identify a nullable value type
```

Comment:
Should this section include an "Important" note that these operations do not apply for nullable reference types. ( know that you've been very careful to include "value" in "nullable value type" throughout the article. That makes me think it would be redundant. On the other hand, the syntax would seem to imply that there are similarities between `int?` and `string?`.  That's a bad assumption.

What are your thoughts?

Reply:
@BillWagner I've added the note to the end of the section. Though being redundant, I think it doesn't hurt. 

Also, after this article is published, should the issue #9662 be "redirected" to the new article?